app/assets/stylesheets/general.css

```diff
@@ -18,6 +18,10 @@ White: #fff;
   box-sizing: border-box;
 }
 
+.rubik-font {
```

Comment:
You could just body{
font-family: "Rubik", sans-serif;
} 
instead of using the class on your body element